src/main/java/it/gov/pagopa/payhub/auth/mypivot/service/MyPivotUsersService.java

```diff
@@ -0,0 +1,34 @@
+package it.gov.pagopa.payhub.auth.mypivot.service;
+
+import it.gov.pagopa.payhub.auth.mypivot.model.MyPivotUser;
+import it.gov.pagopa.payhub.auth.mypivot.repository.MyPivotUsersRepository;
+import java.util.Optional;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MyPivotUsersService {
+
+  private final MyPivotUsersRepository myPivotUsersRepository;
+
+  public MyPivotUsersService(MyPivotUsersRepository myPivotUsersRepository){
+    this.myPivotUsersRepository = myPivotUsersRepository;
+  }
+
+  public void registerMyPivotUser(String externalUserId, String fiscalCode, String firstName, String lastName, String email) {
```

Comment:
To avoid ambiguity, let's call it mappedExternalUserId
```suggestion
  public void registerMyPivotUser(String mappedExternalUserId, String fiscalCode, String firstName, String lastName, String email) {
```